.fakeApi/index.js

```diff
@@ -47,18 +58,29 @@ module.exports = () => {
         lessons_url: `http://localhost:4000/api/v1/instructors/${id}/lessons`,
         slack_id: faker.random.arrayElement([null, faker.internet.userName()]),
         gear_tracking_id: faker.random.arrayElement([null, faker.random.uuid()]),
+        published_courses: faker.random.number({
+          min: 0,
+          max: 20,
+        }),
         published_lessons: has(overrides, 'publishedLessons')
           ? overrides.publishedLessons
           : faker.random.number({
               min: 0,
-              max: 200,
+              max: 70,
             }),
+        revenue: {
+          total: revenueFactory(1000000, 100000),
+          current: 'mar',
+          jan: revenueFactory(),
+          feb: revenueFactory(),
+          mar: revenueFactory(),
+        }
       }
     }),
 
     lessons: times(lessonCount, index => {
       const id = index
-      const title = faker.lorem.words()
+      const title = faker.company.catchPhrase()
       const instructorId = faker.random.number({
         min: 0,
         max: instructorCount
```

Comment:
Updating fake API. I added `published_courses` and `revenue.total`; @joelhooks would it be ok to add these to the real endpoints? I'm using it in the "Total" section:
![image](https://cloud.githubusercontent.com/assets/5497885/20330681/b0ed28e8-ab5d-11e6-9487-cafe532f1b73.png)